### PR TITLE
Started quiet boot with no verbose

### PR DESCRIPTION
**Pull request by byransays**
This adds various commands into the cmdline.txt file located in the boot directory. This will display the default 3 dot splash screen while removing the console commands.

### CODE REVIEW

**Comment by htruong** [comment]
Thanks for submitting the pull, @byransays -- I haven't gotten time to look at it, but initially it looks good. I will have to test the changes soon and will merge if it works!

File: scripts/customize-image-pi.sh [hunk 1]
```diff
@@ -41,8 +41,10 @@ mark_script_run() {
 house_keeping() {
     # we don't need to resize the root part
     sed -i 's/ init\=.*$//' /boot/cmdline.txt
+    sed -i 's/console=tty1/console=tty3/' /boot/cmdline.txt
     echo "disable_splash=1" >> /boot/config.txt
     echo -e "# Disable the PWR LED.\ndtparam=pwr_led_trigger=none\ndtparam=pwr_led_activelow=off" >> /boot/config.txt
+    echo "logo.nologo splash loglevel=0 vt.global_cursor_default=0" >> /boot/cmdline.txt
```
Comment (htruong) [inline scripts/customize-image-pi.sh]: This doesn't work because it will create a new line on the `/boot/cmdline` file. It needs to be on one line

**Review by htruong — COMMENTED** [review]
I have tested your changes and it doesn't work because of the reason I specified.

**Comment by htruong** [comment]
Sorry for the late reply -- I have been able to edit your script just a little bit to make it work! So everything is good and I was able to get your name in the tree now and credit you as a contributor. Thanks for the pull request, and the research put into it!

**Comment by shivam2702** [comment]
Hi, 
Is it working, and merged as I can still see the verbose on the launch?

**Pull request closed without merging**